lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -2518,23 +3297,32 @@ def model_add_construction(model, construction_name, construction_props = nil)
   # @param intended_surface_type [String] intended surface type
   # @param standards_construction_type [String] standards construction type
   # @param building_category [String] building category
+  # @param wwr_building_type [String] building type used to determine WWR for the PRM baseline model
+  # @param wwr_info [Hash] @Todo - check what this is used for
+  # @param surface [OpenStudio::Model::Surface] OpenStudio surface object, only used for surface specific construction, e.g F/C-factor constructions
   # @return [OpenStudio::Model::Construction] construction object
-  def model_find_and_add_construction(model, climate_zone_set, intended_surface_type, standards_construction_type, building_category)
+  def model_find_and_add_construction(model, climate_zone_set, intended_surface_type, standards_construction_type, building_category, wwr_building_type = nil, wwr_info = {}, surface = nil)
```

Comment:
this method is used quite a bit.  For compatability, it may be better to use option argument instead of defaults, e.g `, wwr_building_type: nil, wwr_info: {}, surface: nil) `